src/plugins/data_source_management/public/management_app/compoenent/experimental_callout/experimental_callout.tsx

```diff
@@ -40,7 +40,7 @@ export const ExperimentalCallout = ({ docLinks }: { docLinks: DocLinksStart }) =
               defaultMessage="To leave feedback, visit "
             />
           }
-          <EuiLink href={docLinks.links.noDocumentation.openSearchForum} target="_blank">
+          <EuiLink href={docLinks.links.opensearch.openSearchForum} target="_blank">
```

Comment:
Please also update the mock in src/plugins/data_source_management/public/mocks.ts